avr/cores/megacommand/memory.h

```diff
@@ -1,7 +1,22 @@
 #ifndef MEMORY_H__
 #define MEMORY_H__
 
-#define BANK1_SYSEX1_DATA_START 0x2200
+#define RX_BUF_SIZE 0x80
+#define TX_BUF_SIZE 0x0C00
```

Comment:
gotcha.
TX_BUF_SIZE is `0x0C00` so the buflen type should be `uint16_t` instead of default `uint8_t`.

Reply:
I've seen the `#if (TX_BUF_SIZE >= 256)` part. not an issue.